pkg/verifier/notation/certstoresbytype.go

```diff
@@ -72,18 +72,18 @@ func newCertStoreByType(confInNewFormat verificationCertStores) (certStores, err
 		s[certStoreType(certstoretype)] = make(map[string][]string)
 		parsedStoreData, ok := storeData.(verificationCertStores)
 		if !ok {
-			return nil, fmt.Errorf("certStores: %s assertion to type verificationCertStores failed", storeData)
+			return nil, fmt.Errorf("configuration is unsupported: %+v", storeData)
```

Comment:
Just double checking , would the final message indicate the error comes from notation verifier?

Reply:
Good call! I checked the full path of the error, there is no explicit indicator of notation verifier. I'll add the notation identifier in `notation.go`.